src/shared/views/ChangeDetection/InstructionsScreen.js

```diff
@@ -143,84 +145,99 @@ export default class CDInstructionsScreen extends React.Component<Props> {
                             { alignSelf: 'center', marginTop: 15 },
                         ]}
                     >
-                        Instructions
+                        {t('instructions')}
                     </Text>
                 </View>
 
                 <ScrollView style={styles.container}>
-                    <Text style={styles.header}>Your task:</Text>
+                    <Text style={styles.header}>{t('your task')}</Text>
                     <Text style={styles.tutParagraph}>
-                        You&apos;re looking for{' '}
-                        <Text style={{ fontWeight: 'bold' }}>
-                            changes in buildings
-                        </Text>
-                        . This acts as a clear indicator for a change in
-                        population size.
+                        <Trans i18nKey="CDInstructionsScreen:lookingFor">
```

Comment:
is this the long version of writing `{t('CDInstructionsScreen:lookingFor')}`. Above you always wrapped the text blocks into the t function. Here it looks differently.

Reply:
Almost. This `Trans` component is used when there are nested components, which you could not include in the `{t('...')}` function. See https://react.i18next.com/latest/trans-component for details